ticdc/troubleshoot-ticdc.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Common Problems and Troubleshooting
+summary: Fix common problems while using TiCDC.
+category: reference
+aliases: ['/docs/dev/reference/tools/ticdc/troubleshoot-ticdc/']
```

Comment:
`aliases` is for redirecting a previous link and does not apply to a newly added file.
```suggestion
```

Reply:
`aliases` removed